build/python/backend/requirements.txt

```diff
@@ -27,8 +27,8 @@ nested-lookup==0.2.22
 numpy==1.22.1
 olefile==0.46
 oletools==0.60.1
-opencv-python==4.6.0.66
-opencv-contrib-python==4.6.0.66
+opencv-python==4.8.1.78
```

Comment:
This wasn't in their report, but it showed up for me when I scanned with `trivy`